helmfile/values/fluentd-wc.yaml.gotmpl

```diff
@@ -363,47 +357,45 @@ extraConfigMaps:
       </buffer>
     </match>
     <match **>
-      @id elasticsearch
-      @type "#{ENV['OUTPUT_TYPE']}"
-      @log_level "#{ENV['OUTPUT_LOG_LEVEL']}"
-      include_tag_key "#{ENV['OUTPUT_INCLUDE_TAG_KEY']}"
-      type_name "#{ENV['OUTPUT_TYPE_NAME']}"
-      hosts "#{ENV['OUTPUT_HOSTS']}"
-      port "#{ENV['OUTPUT_PORT']}"
-      path "#{ENV['OUTPUT_PATH']}"
-      scheme "#{ENV['OUTPUT_SCHEME']}"
-      ssl_verify "#{ENV['OUTPUT_SSL_VERIFY']}"
-      ssl_version "#{ENV['OUTPUT_SSL_VERSION']}"
-      user "#{ENV['OUTPUT_USER']}"
-      password "#{ENV['OUTPUT_PASSWORD']}"
-      log_es_400_reason "#{ENV['OUTPUT_LOG_400_REASON']}"
-      reconnect_on_error "#{ENV['OUTPUT_RECONNECT_ON_ERROR']}"
-      reload_on_failure "#{ENV['OUTPUT_RELOAD_ON_FAILURE']}"
-      reload_connections "#{ENV['OUTPUT_RELOAD_CONNECTIONS']}"
-      request_timeout "#{ENV['OUTPUT_REQUEST_TIMEOUT']}"
+        @id elasticsearch
+        @type "#{ENV['OUTPUT_TYPE']}"
```

Comment:
I can't find where this env variable (`OUTPUT_TYPE`) is changed to `opensearch`. I'm guessing that that's needed.
I would also like you to change the ID of all the output nodes to replace `elasticsearch` to `opensearch`

Reply:
helmfile/upstream/fluentd-elasticsearch/values.yaml

But yes, should be moved to the values files instead!